RELEASING.md

```diff
@@ -0,0 +1,61 @@
+Release Process
+===============
+
+Signing key: https://lgrahl.de/pgp-key.txt
+
+1. Check the code:
+
+   ```bash
+   flake8 .
+   isort -rc -c . || isort -rc -df
+   python setup.py checkdocs
+   py.test
+   ```
+
+2. Set variables:
+
+   ```bash
+   export VERSION={VERSION}
+   export GPG=0482ABA6
+   ```
+
+3. Update version number in ``saltyrtc/__init__.py`` and 
+   ``CHANGELOG.rst``, also update the URL with the corresponding tags.
+
+4. Do a signed commit and signed tag of the release:
+
+  ```bash
+  git add setup.py CHANGELOG.md
```

Comment:
`CHANGELOG.md` should probably be changed to `CHANGELOG.rst`. Also, `setup.py` should probably be replaced with `saltyrtc/__init__.py`.